glib-macros/src/genum_derive.rs

```diff
@@ -112,10 +95,23 @@ pub fn impl_genum(input: &syn::DeriveInput) -> TokenStream {
             }
         }
 
+        impl #crate_ident::translate::TryFromGlib<i32> for #name {
```

Comment:
Why is this needed btw?

Reply:
ErrorDomain::from() -> Option, so it was simpler for the common code generator to also return Option. I could have not implemented TryFromGlib and just inlined the call and unwrap() in FromGlib, but I thought that could also be useful (more useful that FromGlib that may panic)